### DISCUSSION TITLE
Looking to test using Jest and React Testing Library

### DISCUSSION THREAD

**Discussion by leanjunio** [vercel/next.js#14141] [Help]
Hey guys, I'm a beginner with Testing and I'm hoping to implement a Jest / React Testing Library thing going with a Next.js application.

**Comment by mithi** [comment]
Did you checkout the following blog posts? 
- https://dev.to/ericdouglas/how-to-configure-jest-on-a-next-js-project-2ic6
- https://medium.com/frontend-digest/setting-up-testing-library-with-nextjs-a9702cbde32d
